seqr/views/utils/pedigree_info_utils.py

```diff
@@ -163,10 +163,12 @@ def _convert_fam_file_rows_to_json(rows):
         json_record = _parse_row_dict(row_dict, i)
 
         # validate
-        if not json_record.get(JsonConstants.FAMILY_ID_COLUMN):
-            raise ValueError("Family Id not specified in row #%d:\n%s" % (i+1, json_record))
-        if not json_record.get(JsonConstants.INDIVIDUAL_ID_COLUMN):
-            raise ValueError("Individual Id not specified in row #%d:\n%s" % (i+1, json_record))
+        columns = [JsonConstants.FAMILY_ID_COLUMN, JsonConstants.INDIVIDUAL_ID_COLUMN]
+        if required_columns:
+            columns += required_columns
+        missing_cols = [col for col in columns if not json_record.get(col)]
+        if missing_cols:
```

Comment:
It may be neater to change to use the `any` function like `if any(not json_record.get(col) for col in columns)`.

Reply:
no because we use the `missing_cols` list to format the error message so we would need to compute this list anyways